### DISCUSSION TITLE
Alpha is live!

### DISCUSSION THREAD

**Discussion by justalever** [getrailsui/railsui#6] [Announcements]
The initial Alpha version is here, and I'm excited to share it with you all! A bunch of work led up until now, but I'm still figuring things out. While in Alpha, my goal is to improve Rails UI to be the most valuable, so any feedback you have, good or bad, please share. 

I wrote about the launch in greater detail here: [https://railsui.com/updates/railsui-alpha-release](https://railsui.com/updates/railsui-alpha-release).

Some useful links:
- [Home page](https://railsui.com)
- [Documentation](https://railsui.com/docs)
- [Discussions](https://github.com/getrailsui/railsui/discussions)
- [Become an Early supporter](https://railsui.com/pricing)

**Comment by brusacco** [comment]
Any integration with https://storybook.js.org/ ?

**Reply by justalever** [reply]
Not at this time be perhaps in the future. I'll have to dive a bit deeper into storybook 👍 .

**Reply by brusacco** [reply]
Thanks, great project!

**Comment by JetAvia** [comment]
Hi Andy..

First of all, congratulations for this wonderful and useful project and releasing the alpha version of RailsUI. We wish you continued success and good returns from this project in the future.

I'm learning Ruby on Rails and I follow your blogs and tutorials continuously and I benefited a lot from them, and I know how much hard work and effort you put in to provide us with all this and we will always be grateful to you for every piece of information you share with us, no matter how small.

**Reply by justalever** [reply]
Thanks so much for the comment! It means a lot to hear feedback like this. I'm glad I can help out!
